config/config.dist.php

```diff
@@ -1,10 +1,13 @@
 <?php
+
+namespace Imbo\MetadataSearch;
+
 return [
     'eventListeners' => [
         'metadata' => [
             'listener' => 'Imbo\MetadataSearch\EventListener\MetadataOperations',
```

Comment:
Oh, right - I didn't spot that it was a string. Suits me right for trying to reviews while eating breakfast :)